src/nunit.analyzers.tests/UpdateStringFormatToInterpolatableString/UpdateStringFormatToInterpolatableStringAnalyzerTests.cs

```diff
@@ -72,93 +78,93 @@ public void AnalyzeWhenFormattableStringIsUsed(string method)
             RoslynAssert.Valid(this.analyzer, testCode);
         }
 
-        [Test]
-        public void AnalyzeAssertBoolWhenNoArgumentsAreUsed()
+        [TestCaseSource(nameof(AssertAndAssume))]
+        public void AnalyzeAssertBoolWhenNoArgumentsAreUsed(string assertOrAssume)
```

Comment:
Presumably all tests should be renamed to include `AndAssume`?

Reply:
Done.